### PR TITLE
cleanup: error typos in deployment_util.go

### PR DESCRIPTION
**Pull request by AllenZMC**
**What type of PR is this?**
 /kind cleanup

**What this PR does / why we need it**:
 make code kinda more readable

```release-note
 NONE
```

### CODE REVIEW

File: pkg/controller/deployment/util/deployment_util.go [hunk 1]
```diff
@@ -857,7 +857,7 @@ func IsSaturated(deployment *apps.Deployment, rs *apps.ReplicaSet) bool {
 }
 
 // WaitForObservedDeployment polls for deployment to be updated so that deployment.Status.ObservedGeneration >= desiredGeneration.
-// Returns error if polling timesout.
+// Returns error if polling timeout.
```
Comment (draveness) [inline pkg/controller/deployment/util/deployment_util.go]: This does not seem like a typo to me.

Comment (odinuge) [reply pkg/controller/deployment/util/deployment_util.go]: I think the correct is `times out`, ref. https://en.wiktionary.org/wiki/times_out#English

**Pull request closed without merging**